src/supertux/statistics.cpp

```diff
@@ -93,39 +94,37 @@ Statistics::calculate_max_caption_length()
 }
 
 void
-Statistics::serialize_to_squirrel(SquirrelVM& vm) const
+Statistics::serialize_to_squirrel(ssq::Table& table) const
```

Comment:
Maybe `table`could be renamed to a variable that indicates what table we're talking about (it's a table in the user preferences, right?) This is optional though.

Reply:
Probably not needed, because the function doesn't really care what the table is, it just adds a "statistics" table to it.